packages/api-react/src/services/farmer.ts

```diff
@@ -20,6 +20,9 @@ export const apiWithTag = api.enhanceEndpoints({
     'HarvestersSummary',
     'HarvesterPlotsKeysMissing',
     'HarvesterPlotsDuplicates',
+    'MissingSignagePoints',
+    'FilterChallengeStat',
+    'partialStats',
```

Comment:
nit: Can we keep the capitalization consistent? Is there a reason to have this named `partialStats` instead of `PartialStats`?